cluster/environments/azure-multiple-clusters/aks-westus-variables.tf

```diff
@@ -10,6 +10,10 @@ variable "gitops_west_path" {
   type = "string"
 }
 
+variable "gitops_west_url_branch" {
+  type = "string"
+}
```

Comment:
I would suggest adding a default value to keep configurable values in `.tfvars` at minimum.